app-frontend/src/app/pages/library/projects/list/list.html

```diff
@@ -1,55 +1,69 @@
 <div class="row content stack-sm">
   <div class="column-8">
     <div class="library-header">
-      <h1 class="h3">Projects</h1>
+      <h1 class="h3"><i class="icon-project"></i>Projects</h1>
     </div>
-    <form>
-      <div class="form-group all-in-one">
-        <label for="name"><i class="icon-project"></i></label>
-        <input id="name" type="text" class="form-control"
-               placeholder="New project name" ng-model="$ctrl.newProjectName">
-        <button class="btn btn-link"
-                type="button"
-                ng-click="$ctrl.createNewProject($ctrl.newProjectName)">
-          Create
-        </button>
+    <div class="column-11" ng-if="$ctrl.projectList.length === 0">
+      <rf-call-to-action-item title="You haven't created any projects yet">
```

Comment:
The CTA looks good, but it shows up for a moment while the scene list loads. I think it could be smoother adding in a list-loading indicator while that request is in progress instead of showing this CTA.